template/apps/web/package.json

```diff
@@ -15,18 +15,19 @@
   },
   "dependencies": {
     "@hookform/resolvers": "2.9.11",
-    "@mantine/core": "6.0.1",
-    "@mantine/dates": "6.0.1",
-    "@mantine/dropzone": "6.0.1",
-    "@mantine/hooks": "6.0.1",
-    "@mantine/modals": "6.0.1",
-    "@mantine/next": "6.0.1",
-    "@mantine/notifications": "6.0.1",
+    "@mantine/core": "7.1.3",
+    "@mantine/dates": "7.1.3",
+    "@mantine/dropzone": "7.1.3",
+    "@mantine/hooks": "7.1.3",
+    "@mantine/modals": "7.1.3",
+    "@mantine/next": "6.0.21",
+    "@mantine/notifications": "7.1.3",
     "@svgr/webpack": "6.5.1",
     "@tabler/icons-react": "2.10.0",
     "@tanstack/react-table": "8.7.9",
     "app-constants": "workspace:*",
     "axios": "1.3.4",
+    "clsx": "2.0.0",
```

Comment:
we can import `clsx` from '@mantine/core`

Reply:
![image](https://github.com/paralect/ship/assets/81081329/99ca4987-9c8b-4abd-962f-3e715da0b855)

I think we can't

Reply:
Did you check it? You can import clsx from mantine, not cx